src/onnx/onnx.cpp

```diff
@@ -814,6 +814,13 @@ struct onnx_parser
             s.visit([&](auto v) { copy(v, std::back_inserter(op.starts)); });
         }
 
+        if(op.axes.empty())
+        {
+            std::vector<int64_t> axes(args[0]->get_shape().lens().size());
+            std::iota(axes.begin(), axes.end(), int64_t{0});
+            op.axes = axes;
+        }
+
```

Comment:
Do we need to make the change in the tf.cpp file to parse slice?
How about making this change in the function tune_attributes() in the file slice.hpp? Then we can change just one place.

Reply:
We should do this in the onnx parsing because the optimization passes read this axes array so we should have it as "normal" as possible.